### PR TITLE
add alias command

### PR DESCRIPTION
**Pull request by skarim**
Fewer keystrokes by aliasing `gs` to `gh stack`. Adds a helper command that will create (and remove) the alias for you.

**New alias command functionality:**

* Added a new `alias` subcommand (`AliasCmd`) that lets users create or remove a shell alias (default: `gs`) for `gh stack`, installing a wrapper script in `~/.local/bin/`. The command validates alias names, checks for conflicts, and handles removal, with special instructions for Windows. (`cmd/alias.go`)
* Registered the new `alias` command in the root command so it is available as a top-level CLI subcommand. (`cmd/root.go`)
* Updated the root command registration test to include the new `alias` command, ensuring it is listed among available subcommands. (`cmd/root_test.go`)

**Testing and validation:**

* Added comprehensive tests for alias creation, removal, validation, idempotency, and edge cases, ensuring robustness and correct error handling. (`cmd/alias_test.go`)

### CODE REVIEW

File: cmd/alias.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+		cfg.Infof("Automatic alias removal is not supported on Windows.")
+		cfg.Printf("Remove the %s.cmd file from your PATH manually.", name)
+	} else {
+		cfg.Infof("Automatic alias creation is not supported on Windows.")
+		cfg.Printf("You can create the alias manually by adding a batch file or PowerShell function.")
+		cfg.Printf("For example, create a file named %s.cmd on your PATH with:", name)
+		cfg.Printf("  @echo off")
+		cfg.Printf("  gh stack %%*")
+	}
+	return ErrSilent
```
[the remark this answers is not in the document]
Comment (skarim) [reply cmd/alias.go:86]: Intentionally exiting with a non-zero code because the alias wasn't created

File: cmd/alias.go [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+	}
+
+	// Ensure the bin directory exists.
+	if err := os.MkdirAll(binDir, 0o755); err != nil {
+		cfg.Errorf("failed to create directory %s: %s", binDir, err)
+		return ErrSilent
+	}
+
+	// Write the wrapper script.
+	if err := os.WriteFile(scriptPath, []byte(markedWrapperContent), 0o755); err != nil {
```
[the remark this answers is not in the document]
Comment (skarim) [reply cmd/alias.go:119]: Fair point, will address
